docs/advanced-features/react-18/streaming.md

```diff
@@ -0,0 +1,62 @@
+# Streaming SSR (Alpha)
+
+React 18 will include architectural improvements to React server-side rendering (SSR) performance. This means you could use `Suspense` in your React components in streaming SSR mode and React will send render them and send through HTTP streams. React server components is also an experimental feature based on streaming. Next.js will also provide server components related features along with other streaming support through `next/streaming`.
+
+## Enable Streaming SSR
```

Comment:
It would be good to add a high-level description of what it means to turn on streaming (because the term is pretty overloaded and possibly unclear). What actually changes if you set this flag to true?  e.g. if you add Suspense without this flag, Suspense fallbacks will be flushed once and it will be full client-side-rendering. If you turn the flag on, the suspending will actually happen on the server and content / script tags will be sent through as the boundaries resolve.